src/database/seeders/20191001165157-demo-role.js

```diff
@@ -0,0 +1,47 @@
+import helpers from '../../utils/helper';
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => Promise.all([
+    queryInterface.bulkInsert('Roles', [
+      {
+      roleName: 'System Administrator',
+      roleValue: 'admin',
+      createdAt: new Date(),
+      updatedAt: new Date()
+    },
+    {
+      roleName: 'Travel Administrator',
+      roleValue: 'Tadmin',
+      createdAt: new Date(),
+      updatedAt: new Date()
+    },
+    {
+      roleName: 'Travel Team Member',
+      roleValue: 'member',
+      createdAt: new Date(),
+      updatedAt: new Date()
+    },
+    {
+      roleName: 'manager',
+      roleValue: 'manager',
+      createdAt: new Date(),
```

Comment:
Good work, but I suggest this file to be in `migrations` instead of `seeders` so that every time I create tables(`run migrations`) roles can be added along with the tables, without running `seeders` again

Reply:
There is no reason to do that as you  as you are seeing its a seeder file